RM_SUI/RM_SUI/ContentView.swift

```diff
@@ -8,12 +8,165 @@
+                                TextField("Value", text: $addedAdditionalBalanceValue)
+                                    .keyboardType(.numberPad)
+                                
+                                Button {
+                                    let transferValue = Int(addedAdditionalBalanceValue) ?? 0
+                                    if transferValue <= balance && transferValue != 0 {
+                                        newAccount.addToBalance(value: transferValue)
+                                        balance -= transferValue
+                                    }
+                                    addedAdditionalBalanceValue = ""
+                                } label: {
+                                    Text("OK")
+                                }
+                                Button(role: .cancel) {
+                                } label: {
+                                    Text("Cancel")
+                                }
+                            } message: {
+                                Text("From maim to additional")
+                            }
+
+
+                    }
+                    Spacer()
+                }
+                
+                RoundedRectangle(cornerRadius: 20)
+                    .fill(Color.orange)
+                    .offset(x: isOnToggle ? 200 : 0)
+                    .overlay {
+                        Image(systemName: "cart")
+                            .font(.largeTitle)
+                            .foregroundColor(.red)
+                            .offset(x: isOnToggle ? 150 : 0)
+                    }
+            }
+            
+            Toggle(isOn: $isOnToggle) {
+                Text("Show settings")
+                    .font(.title2)
+                    .foregroundColor(.red)
+            }.padding()
         }
         .padding()
+        .animation(.spring(response: 0.5,
+                           dampingFraction: 0.7,
+                           blendDuration: 0.3))
+    }
+}
+
+struct AccountManagement: View {
+    
+    @State var isAddingBalanceAlertShown = false
+    @State var balance = 0
+    @State var addedBalanceValue = ""
+    
+    func addToBalance(value: Int) {
+        balance += value
+    }
+    
+    var body: some View {
+        Text("Additional Account")
+            .font(.title2)
+        Text("\(balance)")
+            .foregroundColor(.red)
+        
+        Button {
+            isAddingBalanceAlertShown = true
+        } label: {
+            Text("Increase balance")
+        }
+        .buttonStyle(.bordered)
+        .alert(Text("Add to Additional balance"),
+               isPresented: $isAddingBalanceAlertShown) {
+            TextField("Value", text: $addedBalanceValue)
+                .keyboardType(.numberPad)
+            
+            Button {
+                balance += Int(addedBalanceValue) ?? 0
+                addedBalanceValue = ""
```

Comment:
```suggestion
                addedBalanceValue = ""
```
тоже в константы как emptyString